src/main/java/competition/IdealElectricalContract.java

```diff
@@ -81,38 +76,36 @@ public DeviceInfo turretEncoder() {
         return new DeviceInfo(0, true);
     }
 
-    public DeviceInfo carouselMotor()
-    {
+    public DeviceInfo carouselMotor() {
         return new DeviceInfo(35, false);
     }
 
-    public boolean isCarouselReady(){
+    public boolean isCarouselReady() {
         return true;
     }
 
-    public boolean isCollectorArmLiftingReady() {
+    public boolean isArmReady() {
         return true;
     }
 
-    public DeviceInfo liftingCollectorArmMotor() {
-        return new DeviceInfo(5, true);
-    }
-
     public boolean isFrontCollectingReady() {
         return true;
     }
 
     public DeviceInfo frontCollectingMotor() {
         return new DeviceInfo(6, true);
     }
+    //i think this might be the same as intakeMotor but im not sure
 
-    public boolean isHangingReady() {
+    public boolean isHangingReady() { //same as climber
```

Comment:
what's the reason for this same as climber comment?

Reply:
in the document, it is called climber but I named by subsystem as hanger